statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2633,9 +2633,10 @@ def plot_diagnostics(self, variable=0, lags=10, fig=None, figsize=None):
         # elements
         resid_nonmissing = resid[~(np.isnan(resid))]
         ax = fig.add_subplot(222)
-        # temporarily disable Deprecation warning, normed -> density
-        # hist needs to use `density` in future when minimum matplotlib has it
-        with warnings.catch_warnings(record=True):
+
+        try:
+            ax.hist(resid_nonmissing, density=True, label='Hist')
```

Comment:
comment describing the version numbers where this is relevant so this can eventually be removed?

Reply:
done.